src/instructlab/model/evaluate.py

```diff
@@ -295,6 +300,11 @@ def launch_server(
     type=click.Path(),
     help="Path where all the MMLU Branch tasks are stored. Needed for running mmlu_branch.",
 )
+@click.option(
+    "--gpus",
+    type=click.INT,
+    help="Number of GPUs to consume for evaluation",
```

Comment:
it looks like this should say it only works with the VLLM serving backend?

Reply:
The intention of the param would be to support any backends that are multi gpu capable.  For llama-cpp it's not clear if args are required to consume multiple gpus.  --tensor-split and --main-gpu seem to be tuning params.

Reply:
either way, changing this option has no effect on the llama-cpp backend, right?

Reply:
I'm doing more testing now with llama-cpp to validate

Reply:
Updated to indicate it's not applicable to llama-cpp.